stan/math/rev/scal/fun/ibeta.hpp

```diff
@@ -25,7 +26,7 @@ namespace stan {
         double bprod = 1;
         while (std::abs(diff) > precision
                && ++k < max_steps
-               && !std::isnan(diff)) {
+               && !is_nan(diff)) {
```

Comment:
I already fixed it and pushed.

> On Sep 1, 2016, at 2:42 AM, Daniel Lee <notifications@github.com> wrote:
> 
> In stan/math/rev/scal/fun/ibeta.hpp:
> 
> > @@ -25,7 +26,7 @@ namespace stan {
> >          double bprod = 1;
> >          while (std::abs(diff) > precision
> >                 && ++k < max_steps
> > -               && !std::isnan(diff)) {
> > +               && !is_nan(diff)) {
> 
> I think it can be replaced. And a for loop looks ok. I'll make those changes.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>